KK_ButtPlugin/ButtplugController.cs

```diff
@@ -8,6 +8,14 @@
 
 namespace KK_ButtPlugin
 {
+    public enum VibrationMode
+    {
+        Off = 0_00,
+        Male = 0_01,
+        Female = 0_10,

```

Comment:
Why is male vs female important for us? Do you have an example where the same H scene has vibration outputs for both the male and the female character and those outputs are different? For example, masturbation scenes only have output for the female, so it should be unambiguous that we have to send that output to whatever vibrator is available. Shouldn't it? I'm confused. Do you have some greater plan in mind that will make this distinction important?

Reply:
It's as I stated in another comment, not all players are Male and wish to be synced purely to what the male character feels.  For improved immersion, if you're a male player you shouldn't feel a female character masturbating, and if you're a female player you typically shouldn't feel anything from service animations.  By allowing the player to set the kind of immersion they want, it makes the plugin more inclusive.

Reply:
Another solution would be to have the device set to sync to Male instead of Female 1/2.  However, the problem right now is all calculations are performed relative to the female animator.  If there was a good equivalent for male to sync to, that would be a better option instead of having to differentiate by modes and this setting.

Reply:
Hmmm.....
Part of my brain is shouting "THEN JUST PULL THE GODDAMN THING OUT!". But I do realize there must be toys where this is logistically hard to do, like strap-on stuff and whatnot.
What I think would be the best solution: a hotkey for turning ButtPlugin on/off on demand. But I doubt there is a way to make that work in both desktop and VR.
Since I like the general sentiment, I'll go with your solution. But I don't like that this basically doubles the complexity as the code is now essentially split into "Female" and "Male" experiences, and I fully expect this to come back and bite us in the butt. Oh well. At least we'll have a plug to soothe it. 😁